### PR TITLE
Blob Storage docs for Umbraco 8 + some moving around

### PR DESCRIPTION
**Pull request by sofietoft**

### CODE REVIEW

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+versionFrom: 7.0.0
+---
+
+# Setup Your Site to use Azure Storage for Media and Image Processor Cache
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md]:
```suggestion
# Setup Your Site to use Azure Blob Storage for Media and Image Processor Cache
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+For Umbraco Cloud sites there are some scenarios when you may want, or need, to consider using Azure Blob Storage for your media.  Particularly if your site contains large amounts of media - about 1GB or more.  Having your site’s media in Azure Storage can also help your deployments complete more quickly and has the potential to positively affect site performance as the Image Processor cache is moved to Azure Storage.  It also allows you to easily serve your media from the Azure CDN.
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md]:
```suggestion
For Umbraco sites there are some scenarios when you may want, or need, to consider using Azure Blob Storage for your media.  Particularly if your site contains large amounts of media.  Having your site’s media in Azure Storage can also help your deployments complete more quickly and has the potential to positively affect site performance as the Image Processor cache is moved to Azure Blob Storage.  It also allows you to easily serve your media from the Azure CDN.
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+These packages are only available via NuGet, so ideally you’ll have your site setup to use Visual Studio. You can copy/paste the `PM>` commands into the Package Manager Console. If you don't see the Package Manager Console window, you can open it from the menu View -> Other Windows -> Package Manager Console.
+
+### File System Provider for Azure Blob Storage
+
+You’ll replace the default FileSystemProvider with the `UmbracoFileSystemProviders.Azure` provider.  We recommend doing this first and verifying it behaves as expected before proceeding with the ImageProcessor setup.
+
+```PM> Install-Package UmbracoFileSystemProviders.Azure```
+
+The package is also available on Our Umbraco [https://our.umbraco.com/projects/collaboration/umbracofilesystemprovidersazure/](https://our.umbraco.com/projects/collaboration/umbracofilesystemprovidersazure/)
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md]:
```suggestion
The package is also available on [Our Umbraco](https://our.umbraco.com/projects/collaboration/umbracofilesystemprovidersazure/)
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+        </handlers>
+      </system.webServer>
+    </location>
+  </configuration>
+```
+
+### Image Processor
+You’ll need to install the following ImageProcessor packages (latest versions recommended):
+
+* ImageProcessor.Web.4.9.3.25 or later
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md]:
```suggestion
* ImageProcessor.Web
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+* ImageProcessor.Web.Config.2.4.1.19 or later
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md]:
```suggestion
* ImageProcessor.Web.Config
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+* ImageProcessor.Web.Plugins.AzureBlobCache.1.4.2.19 or later
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md]:
```suggestion
* ImageProcessor.Web.Plugins.AzureBlobCache
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+Then install the ImageProcessor package that places the cached files in the Azure Blob storage:
+
+```PM> Install-Package ImageProcessor.Web.Plugins.AzureBlobCache```
+
+
+### Configuration
+Once the packages have been installed you need to set your configuration as below.  Some of these may have been set when you installed the ImageProcessor packages.
+
+** Update `~web.config`**
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md]:
```suggestion
**Update `~web.config`**
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+    <service prefix="remote.axd" name="RemoteImageService" type="ImageProcessor.Web.Services.RemoteImageService, ImageProcessor.Web">
+      <settings>
+        <setting key="MaxBytes" value="4194304" />
+        <setting key="Timeout" value="3000" />
+        <setting key="Protocol" value="http" />
+      </settings>
+      <whitelist>
+        <add url="https://[your Azure CDN].vo.msecnd.net/" />
+        <add url="https://[your blob account].blob.core.windows.net/" />
+        <add url="https://[your Umbraco cloud site].s1.umbraco.io" />
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md]:
```suggestion
        <add url="https://[your Umbraco site]" />
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 9]
```diff
@@ -0,0 +1,157 @@
+---
+versionFrom: 8.0.0
+---
+
+# Setup Your Site to use Azure Storage for Media and Image Processor Cache
+For Umbraco Cloud sites there are some scenarios when you may want, or need, to consider using Azure Blob Storage for your media.  Particularly if your site contains large amounts of media - about 1GB or more.  Having your site’s media in Azure Storage can also help your deployments complete more quickly and has the potential to positively affect site performance as the Image Processor cache is moved to Azure Storage.  It also allows you to easily serve your media from the Azure CDN.
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
For Umbraco sites there are some scenarios when you may want, or need, to consider using Azure Blob Storage for your media.  Particularly if your site contains large amounts of media.  Having your site’s media in Azure Blob Storage can also help your deployments complete more quickly and has the potential to positively affect site performance as the Image Processor cache is moved to Azure Blob Storage.  It also allows you to easily serve your media from the Azure CDN.
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 10]
```diff
@@ -0,0 +1,157 @@
+---
+versionFrom: 8.0.0
+---
+
+# Setup Your Site to use Azure Storage for Media and Image Processor Cache
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
# Setup Your Site to use Azure Blob Storage for Media and Image Processor Cache
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+
+These packages are only available via NuGet, so ideally you’ll have your site setup to use Visual Studio. You can copy/paste the `PM>` commands into the Package Manager Console. If you don't see the Package Manager Console window, you can open it from the menu View -> Other Windows -> Package Manager Console.
+
+### Umbraco File System Provider
+
+You'll need to install the `UmbracoFileSystemProviders.Azure` provider.  We recommend doing this first and verifying it behaves as expected before proceeding with the ImageProcessor setup.
+
+Find instructions on how to install the package on the projects GitHub page: [UmbracoFileSystemProviders.Azure](https://github.com/JimBobSquarePants/UmbracoFileSystemProviders.Azure/tree/develop-umbraco-version-8). There are detailed instructions available on the project page, also summarized here.
+
+The package is also available on Our Umbraco [https://our.umbraco.com/projects/collaboration/umbracofilesystemprovidersazure/](https://our.umbraco.com/projects/collaboration/umbracofilesystemprovidersazure/) - make sure you download the correct version for Umbraco 8, which is also specified on the page.
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
The package is also available on [Our Umbraco](https://our.umbraco.com/projects/collaboration/umbracofilesystemprovidersazure/) - make sure you download the correct version for Umbraco 8, which is also specified on the page.
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+
+The following six keys will have been added to the `<appSettings>` your `web.config` file.
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
The following six keys will have been added to the `<appSettings>` in your `web.config` file.
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+  <add key="AzureBlobFileSystem.RootUrl:media" value="https://[myAccountName].blob.core.windows.net/" />
+  <add key="AzureBlobFileSystem.ConnectionString:media" 
+      value="DefaultEndpointsProtocol=https;AccountName=[myAccountName];AccountKey=[myAccountKey]" />
+  <add key="AzureBlobFileSystem.MaxDays:media" value="365" />
+  <add key="AzureBlobFileSystem.UseDefaultRoute:media" value="true" />
+  <add key="AzureBlobFileSystem.UsePrivateContainer:media" value="false" />
+</appSettings>
+```
+
+Make sure to udpate this configuration to match your own setup.
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
Make sure to update this configuration to match your own setup.
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+      </handlers>
+    </system.webServer>
+  </location>
+</configuration>
+```
+
+### Image Processor
+You’ll need to install the following ImageProcessor packages (latest versions recommended):
+
+* ImageProcessor.Web.4.9.3.25 or later
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
* ImageProcessor.Web
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+* ImageProcessor.Web.Config.2.4.1.19 or later
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
* ImageProcessor.Web.Config
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+* ImageProcessor.Web.Plugins.AzureBlobCache.1.4.2.19 or later
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
* ImageProcessor.Web.Plugins.AzureBlobCache
```

File: Extending/FileSystemProviders/Azure-Blob-Storage/index.md [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+    <service prefix="remote.axd" name="RemoteImageService" type="ImageProcessor.Web.Services.RemoteImageService, ImageProcessor.Web">
+      <settings>
+        <setting key="MaxBytes" value="4194304" />
+        <setting key="Timeout" value="3000" />
+        <setting key="Protocol" value="http" />
+      </settings>
+      <whitelist>
+        <add url="https://[your Azure CDN].vo.msecnd.net/" />
+        <add url="https://[your blob account].blob.core.windows.net/" />
+        <add url="https://[your Umbraco cloud site].s1.umbraco.io" />
```
Comment (jmayntzhusen) [inline Extending/FileSystemProviders/Azure-Blob-Storage/index.md]:
```suggestion
        <add url="https://[your Umbraco site]" />
```

File: Umbraco-Cloud/Set-Up/Media/index.md [hunk 18]
```diff
@@ -2,184 +2,17 @@
...
-        <setting key="SourceStorageAccount" value="DefaultEndpointsProtocol=https;AccountName=[myAccountName];AccountKey=[myAccountKey]" />
-        <setting key="SourceBlobContainer" value="media" />
-        <setting key="StreamCachedImage" value="false" />
-      </settings>
-    </cache>
-  </caches>
-</caching>
-```
-The final note here is that setting this up will only make it so new media files are added to Blob Storage, if you already have some media files on your project you should copy the contents of the media folder and upload it all to the media blob you set up. Finally you can delete the media folder locally as it is no longer needed. 
+Setting up Azure Blob Storage requires you to install a few packages and follow a few simple steps which we've outline in throrough guide on how to set it up.
```
Comment (jmayntzhusen) [inline Umbraco-Cloud/Set-Up/Media/index.md]:
```suggestion
Setting up Azure Blob Storage requires you to install a few packages and follow a few simple steps which we've outline in a thorough guide on how to set it up.
```

File: Umbraco-Cloud/Set-Up/Media/index.md [hunk 19]
```diff
@@ -2,184 +2,17 @@
...
 
+In this article you can also find more information, about when using Azure Blob Storage on your Umbraco Cloud project.
```
Comment (jmayntzhusen) [inline Umbraco-Cloud/Set-Up/Media/index.md]: This sentence doesn't make a lot of sense, not sure what you are trying to say? 🙂

Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Media/index.md]:
```suggestion
In this article you will find more specific information about setting up Azure Blob Storage on your Umbraco Cloud environments.
```

**Pull request merged**